mkdocs.yml

```diff
@@ -304,6 +304,7 @@ pages:
         - Release Notes: logagent/releasenotes.md
         - FAQ: logagent/faq.md
       - Mobile App SDKs: agents/mobile.md
+      - Browser SDK: agents/browser.md
```

Comment:
Should we also link it from under https://sematext.com/docs/agents/ in the left nav ?
Should it also be added to that Overview page?

Reply:
Added to the `Overview` page. The `Browser SDK` is already in the left nav in the `Agents` section.